ingestion/src/main/java/feast/ingestion/transform/WriteToStore.java

```diff
@@ -86,6 +86,7 @@ public PDone expand(PCollection<FeatureRow> input) {
     StoreType storeType = getStore().getType();
 
     switch (storeType) {
+      case REDIS_CLUSTER:
```

Comment:
Is there code missing here?

Reply:
No. It just falls through. 
we call `RedisCustomIO.write(getStore()` for both the stores.